test/jest.config.js

```diff
@@ -22,8 +22,4 @@ const customJestConfig = {
   },
 }
 
-if (process.env.CI) {
-  customJestConfig.reporters = [['github-actions', { silent: false }], 'summary']
```

Comment:
It feels like we would still benefit from this line. It shows errors inline on the 'Files Changed' tab

Reply:
Added it back to the base jest config in the monorepo root